Source/WebKit/UIProcess/API/Cocoa/WKWebViewPrivate.h

```diff
@@ -780,6 +780,8 @@ typedef NS_OPTIONS(NSUInteger, WKDisplayCaptureSurfaces) {
 - (void)_dismissContentRelativeChildWindows WK_API_AVAILABLE(macos(10.13.4));
 - (void)_setFrame:(NSRect)rect andScrollBy:(NSSize)offset WK_API_AVAILABLE(macos(10.13.4));
 
+@property (nonatomic, readonly) CGFloat minimumMagnification;
```

Comment:
Let's move this up near the other properties and add availability.

Reply:
Done both in https://github.com/WebKit/WebKit/commit/6669154880571b23b5650ce83e97387bd5465ecd